homework/weather/exercise1.md

```diff
@@ -0,0 +1,68 @@
+# Homework 1 - The Weather Widget
+
+* Example code from lecture 1 can be used as a basis.
+* Make sure you understand what each piece that you do use is doing and pick
+  only the part that are necessary to fulfil these requirements.
+* Read through all the requirements carefully
+* Ask any questions in the course Slack workspace
+
+## Configure initial project (1 / 10 points)
+
+* Create *index.html* 
+  * Must include one JavaScript file (e.g /public/app.js) in document *head*
+  * The JavaScript file must be executed after *DOM* has loaded
+  * Set a *title* (_The Weather Widget_) to the document
+* Specify ALL your dependencies in *package.json*
+* yarn install && yarn build:watch must serve your application on localhost:10001
+* yarn install && yarn test must lint your source files (including test/ folder) and run unit tests
```

Comment:
as we agreed we will use `npm` instead

Reply:
Could you share why you want to change from `yarn` to `npm`. I've found `yarn` to be much nicer to work with, especially when you have to install dependencies from 20 * 7 homework submissions 7.

Reply:
Also, the change would impose very many changes across both slides and markdown documentation. I would advise against this unless there is a good reason.

Reply:
Thanks for your comment! 
I totally agree that yarn is faster, and definitely better for such amount of packages 👍